src/streamlink/plugins/chzzk.py

```diff
@@ -199,7 +238,13 @@ def _get_streams(self):
         if self.matches["live"]:
             return self._get_live(self.match["channel_id"])
         elif self.matches["video"]:
-            return self._get_video(self.match["video_id"])
+            return self._get_vod_playback(
+                *self._api.get_videos(self.match["video_id"]),
+            )
+        elif self.matches["clip"]:
+            return self._get_vod_playback(
+                *self._api.get_clips(self.match["clip_id"]),
+            )
```

Comment:
I'd prefer not making the API method calls in `_get_streams()`, as `_get_live()` also doesn't do this. Please add separate methods below `_get_live()`: `_get_video()` / `_get_clip()`